frame/pablo/src/liquidity_bootstrapping.rs

```diff
@@ -76,7 +76,7 @@ impl<T: Config> LiquidityBootstrapping<T> {
 				Ok(pool_id)
 			})?;
 
-		Ok(pool_id)
+		Ok((pool_id, T::AssetId::default()))
```

Comment:
not sure, can you leave TODO here to make use of currency-factory?

Reply:
This is just a placeholder value to for LBP, which does not really have an LP token. But you are right in that I need to skip this value when creating the staking config